test/AndroidManifest.xml

```diff
@@ -11,7 +11,6 @@
         android:targetPackage="org.sqldroid" />
 
     <application
-        android:icon="@drawable/ic_launcher"
```

Comment:
OK. I will merge the pull request later today.

Thanks,

-- Damodar

On Tue, Jun 14, 2016 at 11:32 PM, Uwe Kubosch <notifications@github.com>
wrote:

> In test/AndroidManifest.xml
> <https://github.com/SQLDroid/SQLDroid/pull/65#discussion_r67106549>:
>
> > @@ -11,7 +11,6 @@
> >          android:targetPackage="org.sqldroid" />
> >
> >      <application
> > -        android:icon="@drawable/ic_launcher"
>
> It is the test project, so I guess leaving out the icon is the best.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/SQLDroid/SQLDroid/pull/65/files/56347a2318f54335e2fec295c2a5b34dc8ab50eb#r67106549>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABJJCy5JhHtyLG02Ptj5FzjA6zMiBtYrks5qL5yRgaJpZM4IvSoG>
> .
>